code-of-conduct.md

```diff
@@ -0,0 +1,79 @@
+
+Event staff will also provide support to victims, including, but not limited to: Providing an Escort, Contacting Venue security or local law encforcement, and briefing key event staff for response/victim assistance. In addition, staff will assist those experiencing harassment to ensure they feel safe for the duration of the event.
+
+#### Pre-Event Concerns
+
+If you are planning to attend an upcoming event and have concerns regarding another individual who may be present, please contact the Orchestructure organizers at orchestructure.meetup (at) gmail (dot) com. Precautions will be taken to ensure your comfort and safety, including, but not limited to: providing an escort, prepping onsite event staff, keeping victim and harasser from attending the same talks/social events and providing onsite contact cell phone numbers for immediate contact.
+
+### Projects Code of Conduct
+
+As contributors and maintainers of this project, and in the interest of fostering
+an open and welcoming community, we pledge to respect all people who contribute
+through reporting issues, posting feature requests, updating documentation,
+submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free experience for
+everyone, regardless of level of experience, gender, gender identity and expression,
+sexual orientation, disability, personal appearance, body size, race, ethnicity, age,
+religion, or nationality.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery
+* Personal attacks
+* Trolling or insulting/derogatory comments
+* Public or private harassment
+* Publishing other's private information, such as physical or electronic addresses,
+ without explicit permission
+* Other unethical or unprofessional conduct. If you wouldn't do it at work, don't do it here.
+
+Project maintainers have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are not
+aligned to this Code of Conduct. By adopting this Code of Conduct, project maintainers
+commit themselves to fairly and consistently applying these principles to every aspect
+of managing this project. Project maintainers who do not follow or enforce the Code of
+Conduct may be permanently removed from the project team.
+
+This code of conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the Orchestructure project maintainers at <orchestructure.meetup@gmail.com>.
```

Comment:
```suggestion
Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the Orchestructure project maintainers at orchestructure.meetup (at) gmail (dot) com..
```

Just aligning with earlier style